user.js

```diff
@@ -82,10 +82,10 @@ user_pref("browser.aboutConfig.showWarning", false);
 user_pref("_user.js.parrot", "0100 syntax error: the parrot's dead!");
 /* 0102: set START page [SETUP-CHROME]
  * [SETTING] General > Thunderbird Start Page ***/
-user_pref("mailnews.start_page.enabled", false);
+// user_pref("mailnews.start_page.enabled", false);
 /* 0104: set NEWTAB page
  * true=? (default), false=blank page ***/
-user_pref("browser.newtabpage.enabled", false);
+// user_pref("browser.newtabpage.enabled", false);
```

Comment:
I'd vote for keep being aligned with upstream Arkenfox here. Also, all `browser` preferences should be disabled as TB is an email client (opinionated).

Reply:
I dont think this makes sense, but if it makes maintenance easier, okay